cache/src/main/java/com/uber/buckcache/resources/buckcache/BuckCacheResource.java

```diff
@@ -77,6 +78,28 @@ public String getSummary() throws Exception {
     }
   }
 
+  /**
+   * Used for client side to measure download speed.
+   * @param size - in kilobytes
+   * @return byte[] - with size in kilobytes
+   * @throws Exception
+   */
+  @GET
+  @Path("dummy/{size}")
+  @Produces(MediaType.APPLICATION_OCTET_STREAM)
+  public Response getDummyArtifact(@PathParam("size") int size) throws Exception {
+    Random rand = new Random();
+    rand.setSeed(System.currentTimeMillis());
+    try {
+      byte[] bytes = new byte[size*1024];
```

Comment:
We should limit `size` to a reasonable value so that API is not misused (may be 100 MB response-bytes limit).
Also `size * 1024` will overflow at some point.

nit: `size*1024 --> size * 1024`